.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Question 🤔
+about: Usage question or discussion about Teleport.
+labels: "type: question"
+---
+
+<!--
+  To make it easier for us to help you, please include as much useful information as possible.
+
+  Useful Links:
+  - Documentation: https://gravitational.com/teleport/docs/
+
+  - If you're an OSS user and have a general question, please ask in our community forum: https://community.gravitational.com/c/teleport
+
+  - For Pro and Enterprise users, please open a ticket via the Enterprise support portal: https://dashboard.gravitational.com/web/
+
+  Before opening a new issue, please search existing issues: https://github.com/gravitational/teleport/issues
+-->
```

Comment:
How does this show up? When I view the file all this content is invisible - does Github show/promote it elsewhere on the page somehow?

Reply:
It show's up in the Write, but is invisible ( a comment ) in the preview. So it doesn't necessarily need to be deleted.